cpp/modmesh/universe/bezier.hpp

```diff
@@ -521,17 +523,15 @@ using PointPadFp64 = PointPad<double>;
 namespace detail
 {
 
-// TODO: change the layout to be x0, x1, y0, y1, z0, z1
 template <typename T>
 struct Segment3dNamed
 {
-    T x0, y0, z0, x1, y1, z1;
+    T x0, x1, y0, y1, z0, z1;
```

Comment:
Reorder the field layout to order by coordinate axis instead of by point.

This aligns with `Bezier3d`.